blocks/rss-alexa-feature-block/features/rss/xml.js

```diff
@@ -0,0 +1,161 @@
+import PropTypes from 'fusion:prop-types'
+import Consumer from 'fusion:consumer'
+import get from 'lodash/get'
+import moment from 'moment'
+import getProperties from 'fusion:properties'
+import { resizerKey } from 'fusion:environment'
+import { BuildContent } from '@wpmedia/feeds-content-elements'
+import { generatePropsForFeed } from '@wpmedia/feeds-prop-types'
+import { buildResizerURL } from '@wpmedia/feeds-resizer'
+
+const jmespath = require('jmespath')
+
+const cheerio = require('cheerio')
+
+const getURLType = (url) => {
+  const imgRegex = '/(jpe?g|png|webp|PNG|JPG|WEBP)$/'
+  const audioRegex = '^(https?|ftp|file)://(www.)?(.*?).(mp3)$'
+  const videoRegex = '^(https?|ftp|file)://(www.)?(.*?).(mp4|ts)$'
+  const uri = url.split('.')
+  const type = uri[uri.length - 1]
+  const defaultType = { label: 'Audio', type: `audio/${type}` }
+  if (new RegExp(imgRegex).test(url)) {
+    return { label: 'image', type: `image/${type}` }
+  }
+  if (new RegExp(audioRegex).test(url)) {
+    return { label: 'Audio', type: `audio/${type}` }
+  }
+  if (new RegExp(videoRegex).test(url)) {
+    return { label: 'Video', type: `video/${type}` }
+  }
+
+  return defaultType
+}
+
+const rssTemplate = (
+  elements,
+  {
+    audioAvailable,
+    channelTitle,
+    channelDescription,
+    channelCopyright,
+    channelTTL,
+    channelCategory,
+    channelLogo,
+    itemTitle,
+    pubDate,
+    itemCategory,
+    includeContent,
+    resizerURL,
+    resizerWidth,
+    resizerHeight,
+    domain,
+    feedTitle,
+    feedLanguage,
+    rssBuildContent,
+  },
+) => ({
+  rss: {
+    '@version': '2.0',
+    channel: {
+      ...(channelTitle && { title: channelTitle }),
+      link: `${domain}`,
+      ...(channelDescription && { description: channelDescription }),
+      lastBuildDate: moment
+        .utc(new Date())
+        .format('ddd, DD MMM YYYY HH:mm:ss ZZ'),
+      ...(feedLanguage && { language: feedLanguage }),
+      ...(channelCategory && { category: channelCategory }),
+      ...(channelCopyright && {
+        copyright: channelCopyright,
+      }), // TODO Add default logic
+      ...(channelTTL && { ttl: channelTTL }),
+
+      ...(channelLogo && {
+        image: {
+          url: buildResizerURL(channelLogo, resizerKey, resizerURL),
+          title: channelTitle || feedTitle,
+          link: domain,
+        },
+      }),
+
+      item: elements.map((s) => {
+        let category
+        const enclosureurl = jmespath.search(s, audioAvailable)
+        const url = `${domain}${s.website_url || s.canonical_url || ''}`
+        return {
+          title: jmespath.search(s, itemTitle) || '',
+          link: url,
+          guid: s._id,
+          pubDate: s[pubDate],
+          ...(enclosureurl && {
+            enclosure: {
+              '@url': enclosureurl,
+              '@label': getURLType(enclosureurl).label,
+              '@group': getURLType(enclosureurl).label,
```

Comment:
Is group a valid attribute for an enclosure?  Where did you find this referenced?